src/components/Thresholds/ThresholdFormSection.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useCallback } from 'react';
+
+import { InlineFieldRow, InlineField, Input, useStyles2 } from '@grafana/ui';
+
+import { config } from '@grafana/runtime';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { ThresholdValues } from 'contexts/SuccessRateContext';
+
+interface LabelProps {
+  color: string;
+  title: string;
+}
+
+interface ThresholdSectionProps {
+  label: string;
+  unit: '%' | 'ms';
+  description: string;
+  thresholds: ThresholdValues;
+  setThresholds: (threshold: ThresholdValues) => void;
+}
+
+const getDotStyles = (color: string) => (theme: GrafanaTheme2) => ({
+  container: css`
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    margin-bottom: 5px;
+    margin-top: 5px;
+    width: 80px;
+  `,
+  dot: css`
+    background-color: ${color};
+    width: 15px;
+    height: 15px;
+    border-radius: 50%;
+    margin-right: 10px;
+  `,
+});
+
+const getSectionStyles = () => (theme: GrafanaTheme2) => ({
+  estimate: css`
+    margin-bottom: 0px;
+    font-style: italic;
+    align-self: center;
+    font-size: 0.7rem;
+  `,
+  disabled: css`
+    // This is a hack, can't seem to get the disabled prop on Input to work
```

Comment:
I think you can get the disabled state to work by wrapping the `<Input />` in a `<Field  disabled={disabled} />` component. I don't _think_ that `Field` has to exist in a form, so you should be able to do it stand-alone. See [here](https://github.com/grafana/synthetic-monitoring-app/blob/efde3227caa8784f7e3677de1ef7a5eec57b3140/src/components/CheckTarget.tsx#L67)

Reply:
Nice, will try